_posts/2016-08-27-mouse-dna-magnetic-fields.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: Low magnetic fields leave mouse DNA undamaged
+author: RaoOfPhysics
+image:
+  feature: 2016-08-27-mouse-dna-magnetic-fields.jpg
+  credit: Nogas1974
+  caption: “DNA com GGN”
+---
+
+_Cells in brain, kidney and liver, which are involved in the transport, reabsorption or storage of iron, withstand sustained exposure to magnetic fields._
+
+The 19th-century works of Hans Christian Ørsted, Michael Faraday, James Clerk Maxwell and others demonstrated that electricity and magnetism are in fact two sides of the same coin: a magnet moving near a coil of wires induces a current in the coil and circulating electric current generates magnetic fields.
+We have since harnessed this unified electromagnetism to provide for the energy needs of billions around the globe.
+However, it is thought that prolonged exposure to ambient electromagnetic (EM) fields can affect certain cells in the body, causing a variety of cancers.
+For this reason, several countries have adopted limits for EM exposure, with most European countries limiting fields to 0.1&nbsp;mT ([milliteslas](https://en.wikipedia.org/wiki/Tesla_(unit))).
+
+Researchers tested the effects of EM fields on biological cells in a controlled environment by exposing two groups of mice to low-frequency fields of 0.1&nbsp;mT and 1.0&nbsp;mT respectively over eight weeks, while a third group of mice living in similar conditions but without the EM exposure served as control.
+The frequency chosen for the research was 50&nbsp;Hz, which is the frequency in most countries of the transmission of electricity via power lines.
```

Comment:
Again with the passive voice.
The electromagnetic fields in the experiment had a frequency of 50 Hz, which is ...